newIDE/app/src/SceneEditor/SwipeableDrawerEditorsDisplay/index.js

```diff
@@ -0,0 +1,414 @@
+                      onGetInstanceSize={getInstanceSize}
+                      ref={instancesPropertiesEditorRef}
+                      unsavedChanges={props.unsavedChanges}
+                      historyHandler={props.historyHandler}
+                    />
+                  )}
+                </I18n>
+              )}
+              {selectedEditorId === 'object-groups-list' && (
+                <I18n>
+                  {({ i18n }) => (
+                    <ObjectGroupsList
+                      ref={objectGroupsListRef}
+                      globalObjectGroups={project.getObjectGroups()}
+                      objectGroups={layout.getObjectGroups()}
+                      onEditGroup={props.onEditObjectGroup}
+                      onDeleteGroup={props.onDeleteObjectGroup}
+                      onRenameGroup={props.onRenameObjectGroup}
+                      canRenameGroup={(newName, global) =>
+                        props.canRenameObjectGroup(newName, global, i18n)
+                      }
+                      beforeSetAsGlobalGroup={groupName =>
+                        props.canObjectOrGroupBeGlobal(i18n, groupName)
+                      }
+                      unsavedChanges={props.unsavedChanges}
+                    />
+                  )}
+                </I18n>
+              )}
+              {selectedEditorId === 'instances-list' && (
+                <Paper
+                  background="medium"
+                  square
+                  style={styles.instancesListContainer}
+                >
+                  <InstancesList
+                    instances={initialInstances}
+                    selectedInstances={selectedInstances}
+                    onSelectInstances={selectInstances}
+                    ref={instancesListRef}
+                  />
+                </Paper>
+              )}
+              {selectedEditorId === 'layers-list' && (
+                <LayersList
+                  project={project}
+                  selectedLayer={selectedLayer}
+                  onSelectLayer={props.onSelectLayer}
+                  onEditLayerEffects={props.editLayerEffects}
+                  onEditLayer={props.editLayer}
+                  onRemoveLayer={props.onRemoveLayer}
+                  onRenameLayer={props.onRenameLayer}
+                  onCreateLayer={forceUpdateInstancesPropertiesEditor}
+                  layersContainer={layout}
+                  unsavedChanges={props.unsavedChanges}
+                  ref={layersListRef}
+                  hotReloadPreviewButtonProps={
+                    props.hotReloadPreviewButtonProps
+                  }
+                />
+              )}
+            </SwipeableDrawer>
+            <BottomToolbar
+              selectedEditorId={
+                drawerOpeningState === 'closed' ? null : selectedEditorId
+              }
+              selectEditor={halfOpenOrCloseDrawerOnEditor}
```

Comment:
should this prop be `onSelectEditor` to be more specific, it's kind of a side-effect

Reply:
Yes you're right, I'll rename it